pkg/cmd/cli/cmd/set/set.go

```diff
@@ -57,3 +59,29 @@ func NewCmdSet(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 	templates.ActsAsRootCommand(set, []string{"options"}, groups...)
 	return set
 }
+
+const (
+	setImageLong = `
+Update existing container image(s) of resources.
+`
+
+	setImageExample = `  # Set a deployment configs's nginx container image to 'nginx:1.9.1', and its busybox container image to 'busybox'.
```

Comment:
I wonder how these descriptions will look along the ones produced from `dedent.Dedent` upstream uses. It would be nice to start using the same in both places, maybe.